### DISCUSSION TITLE
Gerry, could you take a look at my Slinger log and see why my Slingerplayer no longer streams?

### DISCUSSION THREAD

**Discussion by shoobydoodoo** [GerryDazoo/Slinger#603] [Q&A]
Hi Gerry, I hope you don't mind I'm addressing this post directly to you because it seems only you are answering these type of questions. 

I have my Slingbox in Taiwan and Slinger server in Los Angeles, following instructions on figure 6 in the "Server Networking Guide." It was working fine until yesterday when the Slingerplayer could no longer stream. There has been a rainstorm around Asia Pacific. I don't know if that caused a temporary blackout, but I can see the Slingbox and router are both back online. If you could take a look at my server log and give me some hints as to what might went wrong, I'd be very very grateful. 

The SB profile I'm trying to connect is "cldad." Please ignore the rest of the profiles as I have a habit of turning some SBs off when not in use. Here's the log. Thanks a million, Gerry! 

`Version : 4.01 Running on Windows-10-10.0.19045-SP0 pid= 5724 C:\Users\raven\OneDrive\桌面\Slingbox\slingbox_server.exe
Using config file config.ini
Connection Manager Running on port 65432 with 10 max streams using URL slingbox.
BOXES [('sb1', 'ustuner'), ('sb2', 'mod'), ('sb3', 'cldad'), ('sb4', 'clmom'), ('sb5', 'ustunerlow'), ('sb6', 'modlow'), ('sb7', 'cldadlow'), ('sb8', 'ustunerfm'), ('sb9', 'modfm'), ('sb10', 'cldadfm'), ('sb11', 'ustunermfi'), ('sb12', 'ustunermin'), ('sb13', 'modmin'), ('sb14', 'cldadmin')]
BOX URL /slingbox/ustuner
Building page for ustuner
Reading Custom Remote definition from remotetuner.txt
BOX URL /slingbox/mod
Building page for mod
Streamer Running:  10 config.ini ustuner ustuner 65432 8388608
Reading Custom Remote definition from remotemod.txt
ustuner Checking for slingbox at ('192.168.86.254', 5001) 0
BOX URL /slingbox/cldad
Streamer Running:  10 config.ini mod mod 65432 8388608
Building page for cldad
mod Checking for slingbox at ('slingtw.ddns.net', 5002) 0
Reading Custom Remote definition from remotecl.txt
BOX URL /slingbox/clmom
Building page for clmom
Using built in default remote page definition.
Streamer Running:  10 config.ini cldad cldad 65432 8388608
cldad Checking for slingbox at ('slingtw.ddns.net', 5002) 0
BOX URL /slingbox/ustunerlow
Building page for ustunerlow
Reading Custom Remote definition from remotetuner.txt
Streamer Running:  10 config.ini clmom clmom 65432 8388608
clmom Checking for slingbox at ('slingtw.ddns.net', 5001) 0
BOX URL /slingbox/modlow
Streamer Running:  10 config.ini ustunerlow ustunerlow 65432 8388608
Building page for modlow
Reading Custom Remote definition from remotemod.txt
ustunerlow Checking for slingbox at ('192.168.86.254', 5001) 0
BOX URL /slingbox/cldadlow
Building page for cldadlow
Reading Custom Remote definition from remotecl.txt
Streamer Running:  10 config.ini modlow modlow 65432 8388608
modlow Checking for slingbox at ('slingtw.ddns.net', 5002) 0
BOX URL /slingbox/ustunerfm
Building page for ustunerfm
Streamer Running:  10 config.ini cldadlow cldadlow 65432 8388608
Reading Custom Remote definition from remotetuner.txt
cldadlow Checking for slingbox at ('slingtw.ddns.net', 5002) 0
BOX URL /slingbox/modfm
Streamer Running:  10 config.ini ustunerfm ustunerfm 65432 8388608
Building page for modfm
Reading Custom Remote definition from remotemod.txt
mod ('slingtw.ddns.net', 5002) OK
mod Using slingbox at  ('slingtw.ddns.net', 5002)
cldad ('slingtw.ddns.net', 5002) OK
cldad Using slingbox at  ('slingtw.ddns.net', 5002)
ustunerfm Checking for slingbox at ('192.168.86.254', 5001) 0
Streamer Running:  10 config.ini modfm modfm 65432 8388608
BOX URL /slingbox/cldadfm
modfm Checking for slingbox at ('slingtw.ddns.net', 5002) 0
modlow ('slingtw.ddns.net', 5002) OK
modlow Using slingbox at  ('slingtw.ddns.net', 5002)
Streamer:  mod Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Streamer:  cldad Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
cldadlow ('slingtw.ddns.net', 5002) OK
cldadlow Using slingbox at  ('slingtw.ddns.net', 5002)
Building page for cldadfm
Reading Custom Remote definition from remotecl.txt
Streamer:  modlow Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Streamer:  cldadlow Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Streamer Running:  10 config.ini cldadfm cldadfm 65432 8388608
BOX URL /slingbox/ustunermfi
cldadfm Checking for slingbox at ('slingtw.ddns.net', 5002) 0
Building page for ustunermfi
Reading Custom Remote definition from remotetuner.txt
modfm ('slingtw.ddns.net', 5002) OK
modfm Using slingbox at  ('slingtw.ddns.net', 5002)
Streamer:  modfm Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
BOX URL /slingbox/ustunermin
Streamer Running:  10 config.ini ustunermfi ustunermfi 65432 8388608
Building page for ustunermin
Reading Custom Remote definition from remotetuner.txt
ustunermfi Checking for slingbox at ('192.168.86.254', 5001) 0
BOX URL /slingbox/modmin
Streamer Running:  10 config.ini ustunermin ustunermin 65432 8388608
Building page for modmin
Reading Custom Remote definition from remotemod.txt
ustunermin Checking for slingbox at ('192.168.86.254', 5001) 0
BOX URL /slingbox/cldadmin
Building page for cldadmin
cldadfm ('slingtw.ddns.net', 5002) OK
cldadfm Using slingbox at  ('slingtw.ddns.net', 5002)
Streamer Running:  10 config.ini modmin modmin 65432 8388608
Reading Custom Remote definition from remotecl.txt
Streamer:  cldadfm Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
modmin Checking for slingbox at ('slingtw.ddns.net', 5002) 0
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
starting up on  port 65432 8388608
Streamer Running:  10 config.ini cldadmin cldadmin 65432 8388608
cldadmin Checking for slingbox at ('slingtw.ddns.net', 5002) 0
modmin ('slingtw.ddns.net', 5002) OK
modmin Using slingbox at  ('slingtw.ddns.net', 5002)
Streamer:  modmin Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
cldadmin ('slingtw.ddns.net', 5002) OK
cldadmin Using slingbox at  ('slingtw.ddns.net', 5002)
Streamer:  cldadmin Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Broadcast Responder Running
Server IP address is 192.168.86.6
ustuner Error connecting to  ('192.168.86.254', 5001)
clmom Error connecting to  ('slingtw.ddns.net', 5001)
clmom Using slingbox at  ('slingtw.ddns.net', 5001)
Streamer:  clmom Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
ustunerlow Error connecting to  ('192.168.86.254', 5001)
ustunerfm Error connecting to  ('192.168.86.254', 5001)
ustunermfi Error connecting to  ('192.168.86.254', 5001)
ustunermin Error connecting to  ('192.168.86.254', 5001)
ustuner No valid slingbox ip info found in config.ini
ustuner Finding Slingboxes on local network. My IP Info =  192.168.86.6
ustunerlow No valid slingbox ip info found in config.ini
ustunerlow Finding Slingboxes on local network. My IP Info =  192.168.86.6
ustunerfm No valid slingbox ip info found in config.ini
ustunerfm Finding Slingboxes on local network. My IP Info =  192.168.86.6
ustunermfi No valid slingbox ip info found in config.ini
ustunermfi Finding Slingboxes on local network. My IP Info =  192.168.86.6
ustunermin No valid slingbox ip info found in config.ini
ustunermin Finding Slingboxes on local network. My IP Info =  192.168.86.6
ustuner Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
ustunerlow Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
ustunerfm Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
ustunermfi Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
ustunermin Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
ustuner Giving up. Sorry...
ustunerlow Giving up. Sorry...
ustunerfm Giving up. Sorry...
ustunermfi Giving up. Sorry...
ustunermin Giving up. Sorry...
06/02/2023, 15:55:43.436   Streaming request from ('192.168.86.63', 56366)  /slingbox/cldad
STREAM=192.168.86.63:56366:0

Slinginfo  "Solo/Pro/ProHD" 12 30 slingtw.ddns.net 5002 3000 10 78904321
cldad Starting Stream for  192.168.86.63:56366
cldad Opening Control stream 0x2000 0 0
Connecting... ('slingtw.ddns.net', 5002) Control
cmd: 0x67 err: 0x34 8
cldad Unknown Error Starting Session. Cant Continue.
cldad ERROR: Slingbox session startup failed.
cldad Logging Out
Streamer:  cldad Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
`

**Comment by GerryDazoo** [comment]
Is the Slingbox in question a Pro-HD? Looks like it's been bricked. Is the router an ASUS? Do you have remote access to the router, check that the config blocking TCP port 5678 outgoing is still in place.

**Reply by shoobydoodoo** [reply]
Yes the SB is a Pro-HD. The router is D-Link. I didn't setup remote router access because I read that increases security risk. It's a modem/router combo from the ISP. I wonder if they reset some settings. It'll be some time before I can check the administrator page. 

It never crossed my mind that it could've been bricked, because I have another Pro-HD for which I got the return message "Slingbox might be bricked." This one just says "Unknown error." 

I guess I'll have to wait to check the port block settings and go from there. Hopefully, you'll be around when I update my finding and I'm in need of further instructions. Thank you, Gerry.
